docs/language-usage.rst

```diff
@@ -0,0 +1,279 @@
+     >>> document.doc_type == language.Document.PLAIN_TEXT
+     True
+
+and the :meth:`~gcloud.language.client.Client.document_from_uri`
+method. In either case, the document type can be specified with
+the ``doc_type`` argument:
+
+  .. code-block:: python
+
+     >>> gcs_url = 'gs://my-text-bucket/sentiment-me.txt'
+     >>> document = client.document_from_uri(
+     ...     gcs_url, doc_type=language.Document.HTML)
+     >>> document.gcs_url == gcs_url
+     True
+     >>> document.doc_type == language.Document.HTML
+     True
+
+.. _analyzeEntities: https://cloud.google.com/natural-language/reference/rest/v1beta1/documents/analyzeEntities
+.. _analyzeSentiment: https://cloud.google.com/natural-language/reference/rest/v1beta1/documents/analyzeSentiment
+.. _annotateText: https://cloud.google.com/natural-language/reference/rest/v1beta1/documents/annotateText
+.. _Document: https://cloud.google.com/natural-language/reference/rest/v1beta1/Document
+.. _Google Cloud Storage: https://cloud.google.com/storage/
+
+Analyze Entities
+----------------
+
+The :meth:`~gcloud.language.document.Document.analyze_entities` method
+finds named entities (i.e. proper names) in the text and returns them
+as a :class:`list` of :class:`~gcloud.language.entity.Entity` objects.
+Each entity has a corresponding type, salience (prominence), associated
+metadata and other properties.
+
+  .. code-block:: python
+
+     >>> text_content = ("Michelangelo Caravaggio, Italian painter, is "
+     ...                 "known for 'The Calling of Saint Matthew'.")
+     >>> document = client.document(text_content)
+     >>> entities = document.analyze_entities()
+     >>> for entity in entities:
+     ...     print('=' * 20)
+     ...     print('    name: %s' % (entity.name,))
+     ...     print('    type: %s' % (entity.entity_type,))
+     ...     print('metadata: %s' % (entity.metadata,))
+     ...     print('salience: %s' % (entity.salience,))
+     ====================
+         name: Michelangelo Caravaggio
+         type: PERSON
+     metadata: {'wikipedia_url': 'http://en.wikipedia.org/wiki/Caravaggio'}
+     salience: 0.75942981
+     ====================
+         name: Italian
+         type: LOCATION
+     metadata: {'wikipedia_url': 'http://en.wikipedia.org/wiki/Italy'}
+     salience: 0.20193423
+     ====================
+         name: The Calling of Saint Matthew
+         type: WORK_OF_ART
+     metadata: {'wikipedia_url': 'http://en.wikipedia.org/wiki/index.html?curid=2838808'}
+     salience: 0.03863598
+
+Analyze Sentiment
+-----------------
+
+The :meth:`~gcloud.language.document.Document.analyze_sentiment` method
+analyzes the sentiment of the provided text and returns a
+:class:`~gcloud.language.sentiment.Sentiment`. Currently, this method
+only supports English text.
+
+  .. code-block:: python
+
+     >>> text_content = "Jogging isn't very fun."
+     >>> document = client.document(text_content)
+     >>> sentiment = document.analyze_sentiment()
```

Comment:
is this stored statefully on the document at all? ie, can i make this call and have it cached for next time?